charts/dcmaidbot/templates/secret.yaml

```
@@ -0,0 +1,57 @@
+  labels:
+    app: dcmaidbot
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    release: {{ .Release.Name }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "0"
+    "helm.sh/hook-delete-policy": before-hook-creation
+type: Opaque
+stringData:
+  NUDGE_SECRET: {{ .Values.nudgeSecret | default "auto-generated-nudge-secret" | quote }}
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)